style.css

```diff
@@ -52,7 +52,9 @@
     font-weight: 900;
 }
 
-/* @import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;600;700&display=swap'); */
+*/
+
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;600;700&display=swap');
```

Comment:
switch from `@import` to `<link>`. (i dmmed you about it)

Reply:
also, add a fallback font

```
font-family: 'Quicksand', sans-serif;
```
^ from https://fonts.google.com/specimen/Quicksand?sidebar.open&selection.family=Quicksand:wght@300;400;500;600;700#glyphs